content/en/developers/guide/dogshell-quickly-use-datadog-s-api-from-terminal-shell.md

```diff
@@ -48,11 +48,50 @@ Next, use your `dog` commands to quickly use the Datadog API from your terminal/
 
 If you would rather write the `.dogrc` file yourself, the content of this file should be as follows:
 
+{{< site-region region="us" >}}
 ```text
 [Connection]
 apikey = <DATADOG_API_KEY>
 appkey = <YOUR_APPLICATION_KEY>
+api_host = https://datadoghq.com
 ```
+{{< /site-region >}}
+{{< site-region region="us3" >}}
+```text
+apikey = <DATADOG_API_KEY>
+appkey = <YOUR_APPLICATION_KEY>
+api_host = https://us3.datadoghq.com
+```
+{{< /site-region >}}
+{{< site-region region="us5" >}}
+```text
+apikey = <DATADOG_API_KEY>
+appkey = <YOUR_APPLICATION_KEY>
+api_host = https://us5.datadoghq.com
+{{< /site-region >}}
+```
```

Comment:
```suggestion
```

Reply:
To clarify, my feedback here is that the closing tag for the text is out of place - it should be above the /site-region tag 😄